src/ChunkedStreamDecoder.php

```diff
@@ -87,6 +87,14 @@ protected function iterateBuffer()
         }
 
         if ($this->nextChunkIsLength) {
+            if (substr($this->buffer, 0, 3) === "0\r\n") {
```

Comment:
IMO it would be cleaner to parse this like a normal chunk header in order to support chunk extensions etc. as usual. But I guess this is good as-is because it already fixes a particular bug, so let's get this in first :+1: